### ISSUE TITLE
Crowin?

### ISSUE THREAD

**Issue by Sollace** [FinalForEach/Cosmic-Reach-Localization#74]
I've been on a bit of a translations bend with my own projects, having just set them up on crowdin. Since I heard Cosmic Reach was getting localizations work I thought I'd suggest the idea here too. :3 

I would totally contribute an afrikaans translation.

For those not aware: Crowin is a group translation platform. It lets users collaborate in translating projects into their own languages and has various tools to make the process easier. It's what Mojang uses for Minecraft [here](https://crowdin.com/project/minecraft) and it is absolutely free to use if you have an open source project. You just have to make a project and submit them [this handy little form](https://crowdin.com/page/open-source-project-setup-request)

**Comment by lvllaby** [comment]
I agree, it would make easier to update translation from en_us.json to other language because when en_us.json is updated we cannot see it and we should look manually. Crowdin would make things easier.

**Comment by FinalForEach** [comment]
Problem being is that Cosmic Reach is not entirely open source yet (and when it becomes open source, it might not meet the OSI definition), so it would not be free. I plan on making some tools to make the process easier once the format is more finalized.

**Comment by Sollace** [comment]
Alrighty! Well I look forward to what you end up deciding on doing.

**Comment by realeatham** [comment]
> Problem being is that Cosmic Reach is not entirely open source yet (and when it becomes open source, it might not meet the OSI definition), so it would not be free. I plan on making some tools to make the process easier once the format is more finalized.

**Comment by Spicylemon2623** [comment]
Can this be closed now?

**Comment by Sollace** [comment]
If there's no intention of using Crowdin, sure you can close it.
